samples/valueentity-counter/src/main/java/com/example/Main.java

```diff
@@ -6,25 +6,42 @@
 package com.example;
 
 import com.akkaserverless.javasdk.AkkaServerless;
+import com.example.actions.DoubleCounterAction;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import com.example.domain.Counter;
 
+
+// tag::registration[]
+// tag::registration-value-entity[]
 public final class Main {
 
   private static final Logger LOG = LoggerFactory.getLogger(Main.class);
-
   public static AkkaServerless createAkkaServerless() {
     // The AkkaServerlessFactory automatically registers any generated Actions, Views or Entities,
     // and is kept up-to-date with any changes in your protobuf definitions.
     // If you prefer, you may remove this and manually register these components in a
     // `new AkkaServerless()` instance.
+    // end::registration-value-entity[]
+    return AkkaServerlessFactory.withComponents(
+            Counter::new,
+            DoubleCounterAction::new);
+
+    // end::registration[]
+    /* the comment hack bellow is need to show only the Counter::new in the value entity docs
+    // tag::registration-value-entity[]
     return AkkaServerlessFactory.withComponents(
             Counter::new);
+    // end::registration-value-entity[]
+     */
+    // tag::registration-value-entity[]
+    // tag::registration[]
```

Comment:
This is the hack I found to reuse the same project but only show the Counter when presenting the Value Entity. 

We do have a bit of manual maintenance, but that's ok. If we ever touch the protos, we will have to touch that file as well.

Reply:
that's fine